change/@uifabric-variants-2020-06-16-19-09-00-user-v-satgar-AddingPolyfill.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
something went crazy on the change files. I think you should manually delete the ones that don't apply. seems like you only care about the charting package being published here.